### ISSUE TITLE
OneDrive: "Sign in to turn on offline access to your lists" pointing to *removed* work account

### ISSUE THREAD

**Issue by RGlanzmann** [MicrosoftDocs/OfficeDocs-SharePoint#3699]
Hi Team

As already reported in the community by forty-two other suffering customers in a thread viewed over a thousand times, this BUG is haunting me every single day and it drives me CRAZY.

https://answers.microsoft.com/en-us/msoffice/forum/all/remove-sign-in-to-turn-on-offline-access-to-your/94c46653-75b7-4447-9bc4-0542b7637822

![image](https://user-images.githubusercontent.com/19373076/224625229-5aa805f1-2910-4f80-82aa-e82cb2e6c5dd.png)

Please have a look into this.

Best regards,
Remo Glanzmann
Logol AG
6300 Zug, Switzerland

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1787d668-b65b-8de4-a182-2b22b43e429e
* Version Independent ID: 64957a69-2f4b-3947-d631-ac9c29e6f68b
* Content: [Introduction to SharePoint and OneDrive - SharePoint in Microsoft 365](https://learn.microsoft.com/en-us/sharepoint/introduction)
* Content Source: [SharePoint/SharePointOnline/introduction.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/public/SharePoint/SharePointOnline/introduction.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by RGlanzmann** [comment]
I opened a support case (TrackingID#2303141420000146). They have no clue. Support suggested to just switch off List Sync

•	Navigate to HKLM\SOFTWARE\Policies\Microsoft\OneDrive] "DisableNucleusSync" = "dword:1"

But in the meantime I found the formers employers servers listed in this file:
"C:\Users\remog\AppData\Local\Microsoft\OneDrive\ListSync\Business1\settings\Microsoft.ListSync.db"

As other users already found out, simply deleting the folder seems not to solve the issue, there seems to be a cloud-side reference somewhere as well.

How on earth can we get a developer to properly investigate this issue? 54 users need a solution. The community thread has over 1300 views and counting ...

https://answers.microsoft.com/en-us/msoffice/forum/all/remove-sign-in-to-turn-on-offline-access-to-your/94c46653-75b7-4447-9bc4-0542b7637822
